src/System.CommandLine/Parsing/Parser.cs

```diff
@@ -23,22 +23,22 @@ public Parser(Command command) : this(new CommandLineConfiguration(command))
         }
         
         /// <summary>
-        /// Initializes a new instance of the Parser class with using the default <seealso cref="RootCommand"/>.
+        /// Initializes a new instance of the <see cref="T:System.CommandLine.Parsing.Parser" /> class using the default <see cref="T:System.CommandLine.RootCommand" />.
```

Comment:
In the ///, it's best not to add prefixes to crefs, and also not to fully qualify the type. mdoc will convert it as needed.

```suggestion
        /// Initializes a new instance of the <see cref="Parser" /> class using the default <see cref="RootCommand" />.
```